internal/provider/resource_aci_application_epg.go

```diff
@@ -4211,7 +4197,7 @@ func (r *FvAEPgResource) ImportState(ctx context.Context, req resource.ImportSta
 func getAndSetFvAEPgAttributes(ctx context.Context, diags *diag.Diagnostics, client *client.Client, data *FvAEPgResourceModel) {
 	requestData := DoRestRequest(ctx, diags, client, fmt.Sprintf("api/mo/%s.json?rsp-subtree=children&rsp-subtree-class=%s", data.Id.ValueString(), "fvAEPg,fvCrtrn,fvRsAEPgMonPol,fvRsBd,fvRsCons,fvRsConsIf,fvRsCustQosPol,fvRsDomAtt,fvRsDppPol,fvRsFcPathAtt,fvRsIntraEpg,fvRsNodeAtt,fvRsPathAtt,fvRsProtBy,fvRsProv,fvRsSecInherited,fvRsTrustCtrl,tagAnnotation,tagTag"), "GET", nil)
 
-	*data = *getEmptyFvAEPgResourceModel()
+	readData := getEmptyFvAEPgResourceModel()
```

Comment:
why change this? is there an issue with the current getEmpty function implementation?

Reply:
No, there is no issue with the implementation. This was done to have more flexibility to read data from the state when necessary when we don't rely on data returned by the API.

Reply:
ok